lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -1608,4 +1609,22 @@ class TTIRGenericOpCBs : public impl::TTIRGenericOpCBsBase<TTIRGenericOpCBs> {
   }
 };
 
+class TTIRPrivatizeFunctions
+    : public impl::TTIRPrivatizeFunctionsBase<TTIRPrivatizeFunctions> {
+public:
+  using impl::TTIRPrivatizeFunctionsBase<
+      TTIRPrivatizeFunctions>::TTIRPrivatizeFunctionsBase;
+
+  void runOnOperation() final {
+    ModuleOp module = getOperation();
+    module.walk([](mlir::func::CallOp callOp) {
+      mlir::StringRef calleeName = callOp.getCallee();
+      ModuleOp module = callOp->getParentOfType<ModuleOp>();
```

Comment:
you can avoid this line here by add `[&module]` to lambda signature
```suggestion
      ModuleOp module = callOp->getParentOfType<ModuleOp>();
```
`module.walk([&module](mlir::func::CallOp callOp)`